Dangerfile

```diff
@@ -0,0 +1,28 @@
+# By Jakub Kaspar 26/02/2020
+# MessageKit, 2020
+
+# This runs on CI
+mergeable_state = github.pr_json["mergeable_state"]
+
+# Make it more obvious that a PR a draft
+if mergeable_state == "draft"
+  warn("PR is marked as Draft")
+end
+
+# Mainly to encourage writing up some reasoning about the PR, rather than just leaving a title
+if github.pr_body.length < 5
+  fail("Please provide a summary in the Pull Request description")
+end
+
+if !git.modified_files.include?("CHANGELOG.md") && has_app_changes
+  fail("Please include a CHANGELOG entry. \nYou can find it at [CHANGELOG.md](https://github.com/MessageKit/MessageKit/blob/master/CHANGELOG.md).")
```

Comment:
👍

Reply:
This PR should have a changelog entry (right?)

Reply:
I don't think so.. CI and this kind of stuff is not user related so probably shouldn't make it into release notes?